### PR TITLE
Story/case widgets findings

### PR DESCRIPTION
**Pull request by sofiaIvarsRitense**
- Fix font size for tile title
- Fix long titles in the collection widgets

### CODE REVIEW

File: projects/valtimo/dossier/src/lib/components/dossier-detail/tab/widgets/components/collection/widget-collection.component.scss [hunk 1]
```diff
@@ -14,6 +14,10 @@
  * limitations under the License.
  */
 
+label {
+  max-width: 80vh;
```
Comment (teodor-ritense) [inline projects/valtimo/dossier/src/lib/components/dossier-detail/tab/widgets/components/collection/widget-collection.component.scss]: Why vh? Could you use percent?

Comment (sofiaIvarsRitense) [reply projects/valtimo/dossier/src/lib/components/dossier-detail/tab/widgets/components/collection/widget-collection.component.scss]: I tried it before but if I use percent, the div is not resized

Comment (theo-ritense) [reply projects/valtimo/dossier/src/lib/components/dossier-detail/tab/widgets/components/collection/widget-collection.component.scss]: This does not handle the overflow properly. @mbritense suggested to use this:
```
white-space: nowrap;
overflow: hidden;
text-overflow: ellipsis;
```

File: projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts [hunk 2]
```diff
@@ -92,28 +92,31 @@ export class DossierManagementWidgetWizardComponent {
             secondaryLabel: this.translateService.instant(secondaryLabels[WidgetWizardStep.TYPE]),
           }),
           disabled: editMode,
-          complete: editMode,
+          complete: this.widgetWizardService.selectedWidget()?.type ? true : false,
```
Comment (teodor-ritense) [inline projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts]:
```suggestion
          complete: !!this.widgetWizardService.selectedWidget()?.type,
```

File: projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts [hunk 3]
```diff
@@ -92,28 +92,31 @@ export class DossierManagementWidgetWizardComponent {
...
         },
         {
           label: this.translateService.instant('widgetTabManagement.wizard.steps.width'),
           ...(secondaryLabels[WidgetWizardStep.WIDTH] && {
             secondaryLabel: this.translateService.instant(secondaryLabels[WidgetWizardStep.WIDTH]),
           }),
           disabled: !secondaryLabels[WidgetWizardStep.TYPE],
+          complete: this.widgetWizardService.widgetWidth() ? true : false,
```
Comment (teodor-ritense) [inline projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts]:
```suggestion
          complete: !!this.widgetWizardService.widgetWidth(),
```

File: projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts [hunk 4]
```diff
@@ -92,28 +92,31 @@ export class DossierManagementWidgetWizardComponent {
...
         },
         {
           label: this.translateService.instant('widgetTabManagement.wizard.steps.style'),
           ...(secondaryLabels[WidgetWizardStep.STYLE] && {
             secondaryLabel: this.translateService.instant(secondaryLabels[WidgetWizardStep.STYLE]),
           }),
           disabled: !secondaryLabels[WidgetWizardStep.WIDTH],
+          complete: this.widgetWizardService.widgetStyle() ? true : false,
```
Comment (teodor-ritense) [inline projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts]:
```suggestion
          complete: !!this.widgetWizardService.widgetStyle(),
```

File: projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts [hunk 5]
```diff
@@ -92,28 +92,31 @@ export class DossierManagementWidgetWizardComponent {
...
         },
         {
           label: this.translateService.instant('widgetTabManagement.wizard.steps.content'),
           disabled:
             !secondaryLabels[WidgetWizardStep.TYPE] ||
             !secondaryLabels[WidgetWizardStep.WIDTH] ||
             !secondaryLabels[WidgetWizardStep.STYLE],
+          complete: this.widgetWizardService.widgetContent() ? true : false,
```
Comment (teodor-ritense) [inline projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.ts]:
```suggestion
          complete: !!this.widgetWizardService.widgetContent(),
```

File: projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.html [hunk 6]
```diff
@@ -82,14 +82,16 @@ <h3 cdsModalHeaderHeading>
...
-      [disabled]="backButtonDisabled()"
-      (click)="onBackButtonClick()"
-    >
-      {{ 'interface.back' | translate }}
-    </button>
+    @if (currentStep() !== WidgetWizardSteps.TYPE) {
+      <button
+        class="valtimo-dossier-management-widget-wizard__back"
+        cdsButton="secondary"
+        [disabled]="backButtonDisabled()"
```
Comment (teodor-ritense) [inline projects/valtimo/dossier-management/src/lib/components/dossier-management-widget-wizard/dossier-management-widget-wizard.component.html:89]: backButtonDisabled() signal needs to be updated to remove the currentStep condition

**Review by teodor-ritense — APPROVED** [review]

**Pull request merged**